### PR TITLE
Fix bad code style.

### LINKED ISSUE #1180: bad code style
https://github.com/apache/incubator-apisix/pull/1167/files#diff-62edadffc237f13dc28a694080d293a8R543

we need a better style, here is an example:

```
local function system_dns_resolver()
    local file, err = io.open("/etc/resolv.conf")
    if err then
        return nil, err
    end

    local dns_addrs = {}
    ... ...

    file:close()
    return dns_addrs
end
```

### PR DESCRIPTION
**Pull request by Akayeshmantha**
### Full changelog

* Fix bad code style in apisix file.

### Issues resolved

Fix  #1180

### CODE REVIEW

File: bin/apisix [hunk 1]
```diff
@@ -540,24 +540,22 @@ local function check_or_version(cur_ver_s, need_ver_s)
...
-            end
+    local dns_addrs = {}
+    for line in file:lines() do
+        local addr, n = line:gsub("^nameserver%s+(%d+%.%d+%.%d+%.%d+)%s*$", "%1")
+        if n == 1 then
+            table.insert(dns_addrs, addr)
         end
-        return dns_addrs
     end
+    return dns_addrs
```
Comment (membphis) [inline bin/apisix]: We better close the file handle before exiting the function. `file:close()`

Comment (moonming) [reply bin/apisix]: the old codes not close file too.
@membphis Do we have test cases to cover `bin/apisix`? only `apisix start/stop` looks not good enough.

Comment (Akayeshmantha) [reply bin/apisix]: Oops missed that @membphis  fixed it  :smiley:

**Comment by membphis** [comment]
@Akayeshmantha merged, many thx

**Pull request merged**